src/components/NumericStepper/NumericStepper.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import styles from './NumericStepper.scss';
+
+import { StepperButton } from '../Buttons';
+
+interface Props {
+    /**
+     * Callback is called when user changes component value by using built-in controls
+     */
+    onChange: (value: number) => void;
+    /**
+     * Increase/decrease step value
+     */
+    step?: number;
+    /**
+     * Minimum possible value of NumericStepper
+     */
+    minValue?: number;
+    /**
+     * Maximum possible value of NumericStepper
+     */
+    maxValue?: number;
+    /**
+     * The value stepping starts from
+     */
+    defaultValue?: number;
+    /**
+     * De-facto CSS property that sets custom width for input
+     */
+    customWidth?: string;
+}
+
+const { block, elem } = bem('NumericStepper', styles);
+
+export const NumericStepper: React.FC<Props> = (props) => {
+    const { onChange, step, minValue, maxValue, defaultValue, customWidth } = props;
+
+    const [currentValue, setCurrentValue] = React.useState<number>(defaultValue || minValue || 0);
+
+    if (
+        step === undefined ||
+        defaultValue === undefined ||
+        minValue === undefined ||
+        maxValue === undefined
+    )
+        return null; // TODO: Remove this redundant "not-undefined" check
```

Comment:
Guys could you please assist me in this case?
I'm not that familiar with TS unit testing and don't get why the linter wants me to do this kind of check when I already have defaultProps.

Reply:
yep, we need the additional checks if even we have `defaultProps`. TypeScript can't infer the type from `defaultProps`.
https://github.com/textkernel/oneui/blob/master/CONTRIBUTING.md#type-definitions-for-props-and-states

Reply:
you can also add the default values when destructure props in the beginning. It helps with the TS check, but has a potential pitfall of later updating the default in one place, but not the other, and so creating some bugs. Not the most convenient...

Reply:
Got it.